src/server/scripts/Commands/cs_guild.cpp

```diff
@@ -180,6 +181,65 @@ class guild_commandscript : public CommandScript
         uint8 newRank = uint8(atoi(rankStr));
         return targetGuild->ChangeMemberRank(targetGuid, newRank);
     }
+
+    static bool HandleGuildInfoCommand(ChatHandler* handler, char const* args)
+    {
+        Player* target;
+        uint32 guildId;
+        std::string guildName;
+        std::string guildMasterName;
+        Guild* guild;
```

Comment:
Having variables at the top is in general not the preferred way in C++.
Please move them as close to where they are used as possible.
Also this may fix travis from complaining as you would do `if (Type* var = GetSomething())` and it would explicitly tell that you are doing an assignment and not possibly some comparing operation.
Also if you need something like guild to be at the top for the code then you should probably initialize it with nullptr when there is no value.

Reply:
Thanks @Rochet2 :)